README.md

```diff
@@ -2,21 +2,23 @@
+1. Explain how the Learn IDE (Integrated Development Environment) mimics the behavior of the shell
 2. Use `pwd` to identify the current directory of your IDE session
 3. Use `ls` to list the files in the current directory of your IDE session
-4. Use `cd` and `cd ..` to change directories in your IDE session. 
+4. Use `cd` and `cd ..` to change directories in your IDE session.
 
 
 ## Overview
 
-When you open a file on your computer, you locate it by navigating through the directories on your computer's file system using Finder. Even files on your Desktop are stored in your computer's file system.
+As a developer, you spend a lot of time on your computer. You've already learned about how the shell allows you to interact with your computer on a more granular level. Right now, when you open a file on your computer, you locate it by navigating through the directories on your computer's file system using Finder (Mac) or File Explorer (Windows). Every file on your computer is stored in your computer's file system, including files on your Desktop.
 
-When you open an application from your Finder or Desktop, it always happens from the context of a "Working Directory" - the directory of your computer you were in when you executed the program. When you click on a file on your Desktop or Open an application from Your Dock or Applications directory, you are still opening a file in a directory. The Dock and Desktop are just abstractions for that directory to make them easy to access.
+When you open an application, it always happens from the context of a "Working Directory" - the directory of your computer you were in when you executed the program. This working directory could be your Desktop, an Application folder, or a hidden directory that you can't see by default. When you click on a file on your Desktop or open an application from Your Dock or Applications directory, you are still opening a file in a directory. The Dock and Desktop are just abstractions for that directory to make them easy to access.
 
-We're used to navigating and operating on these files using our GUI, or Graphical User Interface, provided by OS X (Macintosh) or whatever operating system we're using (e.g. Windows).
+We're used to navigating and operating on these files using our GUI, or Graphical User Interface, provided by any operating system we are using (e.g. OS or Windows).
 
-The shell (or Terminal) in your IDE allows you to access all of the files associated with the code labs you're working through on Learn. This Terminal provides you with a Command Line Interface to navigate and operate on the files and folders in the IDE. This is sometimes faster and easier than using the GUI. 
+The shell (or Terminal) in your IDE allows you to access all of the files associated with the code labs you're working through on Learn. This Terminal provides you with a Command Line Interface to navigate and operate on the files and folders in the IDE.
```

Comment:
Is the consistent phrase "command prompt?" I think you put that in my review yesterday. Let's make a choice and document it in the [style guide](https://github.com/learn-co-curriculum/curriculum-team/blob/master/style_guide.md).